AirCasting/CreateSessionViews/CreateSessionDetails/CreateSessionDetailsViewModel.swift

```diff
@@ -43,7 +51,13 @@ class CreateSessionDetailsViewModel: ObservableObject {
         return sessionContext
     }
     
-    func checkIfWiFi(sessionContext: CreateSessionContext) -> CreateSessionContext {
+    private func saveWifiPassword() {
+        guard !wifiPassword.isEmpty else { return }
+        guard let passwordData = wifiPassword.data(using: .utf8) else { return }
+        try? KeychainManager.save(service: "wifi", account: wifiSSID, password: passwordData)
```

Comment:
keychain service names should be stored in a special-purpose enum. We really don't want to get in trouble by typos here